src/app/shared/components/Breadcrumbs/Containers/BreadcrumbsDriveView.tsx

```diff
@@ -18,6 +21,15 @@ const BreadcrumbsDriveView = (props: BreadcrumbsDriveViewProps) => {
   const { namePath } = props;
   const dispatch = useAppDispatch();
   const { selectedWorkspace } = useAppSelector((state: RootState) => state.workspaces);
+  const currentBreadcrumb = namePath[namePath.length - 1];
+  const { breadcrumbDirtyName } = useDriveItemStoreProps();
+
+  useEffect(() => {
+    if (breadcrumbDirtyName) {
+      getAncestorsAndSetNamePath(currentBreadcrumb.uuid, dispatch);
+      dispatch(uiActions.setCurrentEditingBreadcrumbNameDirty(''));
```

Comment:
If the currentEditingBreadcrumbNameDirty is no longer in use, check if you can remove it and update the breadcrumb during the flow when renaming folders from the breadcrumb.